docs/src/themes.tsx

```diff
@@ -54,6 +70,42 @@ export const lightPalette = {
 };
 
 export const darkPalette = {
```

Comment:
shoudn't this be like `export const darkPalette = {...} as Palette`? So we are always confident we have the same fields on all themes?